### PR TITLE
update app services left-nav

### PR DESCRIPTION
**Pull request by srijitamukherjee**
Signed-off-by: Srijita Mukherjee <srmukher@redhat.com>

Before: 
Screenshot from 2021-09-22 23-53-33

After:
Screenshot from 2021-09-22 23-52-48

### CODE REVIEW

**Comment by srijitamukherjee** [comment]
@pmuir Please review!

**Comment by pmuir** [comment]
Looks great

@karelhala @Hyperkid123 @ryelo please don't merge this until I get the goahead from our Product Management team.

File: chrome/application-services-navigation.json
Comment (Hyperkid123) [inline chrome/application-services-navigation.json:22]: For the database access app to work, you also have to update the `fed-mods.json` and `main.yml` files. There are `dbass` entries in the `ci-beta` branch already.

Comment (srijitamukherjee) [reply chrome/application-services-navigation.json:22]: @Hyperkid123 I have made the changes you suggested! Please have a look!

**Review by Hyperkid123 — APPROVED** [review]

**Comment by pmuir** [comment]
LGTM

**Comment by barnabycourt** [comment]
@karelhala @ryelo are you good with this change? @pmuir Did you get the go-ahead from the Product Management team?

**Comment by tiffanynolan** [comment]
@pmuir @srijitamukherjee - Can this PR be scoped to just the promotion of Service Registry? There is a new, open PR for adding and promoting the Subscriptions info which I think takes care of the rest of the items. See: https://github.com/RedHatInsights/cloud-services-config/pull/968

**Comment by pmuir** [comment]
This PR is out of date. It just adds database access which is already done. We'll want a new one for SR.

**Comment by rblackbu** [comment]
@tiffanynolan my apologies but I am unclear as to which developer(s) and team is taking ownership for stitching all the new nav updates together? Please advise. Thanks, Russ
@cdcabrera @barnabycourt

**Comment by tiffanynolan** [comment]
> @tiffanynolan my apologies but I am unclear as to which developer(s) and team is taking ownership for stitching all the new nav updates together? Please advise. Thanks, Russ @cdcabrera @barnabycourt

This PR is out of date. The correct one for the nav updates is the one that @cdcabrera has been working on. https://github.com/RedHatInsights/cloud-services-config/pull/968

**Comment by tiffanynolan** [comment]
@ryelo @Hyperkid123 - Can one of you close this PR? It's outdated and not needed anymore.

**Pull request closed without merging**
